internal/workspaces/notebooks/notebooks.go

```diff
@@ -202,6 +223,23 @@ func validateNameArg(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func open(ctx context.Context, id string) error {
+	env, err := runtime.NewEnv(ctx)
+	if err != nil {
+		return err
+	}
+
+	token, err := nbtoken.LoadToken(id)
+	if err != nil {
+		// TODO(aya): implement get token API?
+		return err
+	}
+
+	fmt.Println("Opening browser...")
+	nbURL := fmt.Sprintf("%s/services/notebooks/%s?token=%s", env.Config.EndpointURL, id, token)
+	return browser.OpenURL(nbURL)
```

Comment:
It would be nice if we check the status of the notebook or the pod before opening the URL and returning an explicit error if the notebook is still not running.

Reply:
It's same for job, but even when the status is `running`, the image pull can take a long time.. Probably we need to push some more info to Job/Notebook.